db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 # the relevant ENV vars after it has run at least once.
 RolesConversion.new.ncr_budget_approvers
 RolesConversion.new.gsa18f_approvers
+RolesConversion.new.gateway_roles
```

Comment:
benefit of *not* including in seeds is that we run them before each test, so each new record added to seeds is slowing us down quite a bit